### ISSUE TITLE
Build stuck on GCS remote cache

### ISSUE THREAD

**Issue by guoshimin** [bazelbuild/bazel#5112]
### Description of the problem / feature request:

Build gets stuck on GCS remote cache

### Bugs: what's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

It happens sporadically. Not easily reproducible.

### What operating system are you running Bazel on?

Linux

### What's the output of `bazel info release`?

release 0.11.1

### If `bazel info release` returns "development version" or "(@non-git)", tell us how you built Bazel.

> Replace this line with your answer.

### What's the output of `git remote get-url origin ; git rev-parse master ; git rev-parse HEAD` ?

> Replace this line with your answer.

###  Have you found anything relevant by searching the web?

No

### Any other information, logs, or outputs that you want to share?

.bazelrc:
```
build --remote_http_cache=https://storage.googleapis.com/<gcs_bucket>
build --spawn_strategy=remote --genrule_strategy=remote
build --strategy=Javac=remote --strategy=Closure=remote
build --google_credentials=<service_account_credential>
```

```
bazel run //:gazelle
Extracting Bazel installation...
............
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
    currently loading: 
Analyzing: target //:gazelle (2 packages loaded)
Analyzing: target //:gazelle (2 packages loaded)
Analyzing: target //:gazelle (26 packages loaded)
Analyzing: target //:gazelle (27 packages loaded)
Analyzing: target //:gazelle (28 packages loaded)
Analyzing: target //:gazelle (28 packages loaded)
Analyzing: target //:gazelle (28 packages loaded)
INFO: Analysed target //:gazelle (31 packages loaded).
INFO: Found 1 target...
[0 / 2] [-----] BazelWorkspaceStatusAction stable-status.txt
[9 / 35] [-----] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 3s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 8s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 31s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 39s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 66s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 92s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 109s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 147s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 173s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 202s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 235s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 273s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 317s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 368s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 426s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 493s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 571s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 659s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 761s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 879s remote ... (3 actions running)
[9 / 35] GoCompile external/io_bazel_rules_go/go/tools/builders/linux_amd64_stripped/link~/go/tools/builders/link.a [for host]; 1014s remote ... (3 actions running)
```

**Comment by buchgr** [comment]
I would tentatively say this issue sounds like it was fixed by https://github.com/bazelbuild/bazel/commit/04ce86e8ba96630f89a436167b7f3a195c5e50e7 which only landed in 0.12.0. Can you try upgrading to 0.12.0 please?

**Comment by guoshimin** [comment]
Trying 0.12.0. Will reopen if it still happens.
